File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex9.c
#include <stdio.h>

main()
{
    printf("N Termos da Sequencia de Fibonacci\n\n");

    //vars
    int n1=1,n2=1, aux, num, i;

    //Entrada
    printf("Insira a quantidade de termos: ");
    scanf("%d", &num);

    for(i=1; i<=num; i++)
    {
        printf("%d,", n1);
        aux = n1;
        n1 = n2;
        n2 += aux;
    }


}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex9.java
//Importando Scanner
import java.util.Scanner;

public class Ex9{
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Sequência de Fibonacci\n\n");
		
		//vars
		int n1=1, n2=1, aux, num, i;
		
		//Entrada
		System.out.print("Insira a quantidade de termos: ");
		num = ent.nextInt();
		
		//Processamento
		for( i=1; i<=num; i++ )
		{
			System.out.println(n1);
			aux = n2;
			n2 += n1;
			n1 = aux;
		}
		
	}
}